config.js

```diff
@@ -38,16 +38,28 @@ var config = {
     },
   },
   blockchainExplorerOpts: {
-    livenet: {
-      provider: 'insight',
-      url: 'https://insight.bitpay.com:443',
+    btc: {
+      livenet: {
+        provider: 'insight',
+        url: 'https://insight.bitpay.com:443',
+      },
+      testnet: {
+        provider: 'insight',
+        url: 'https://test-insight.bitpay.com:443',
+        // url: 'http://localhost:3001',
+        // Multiple servers (in priority order)
+        // url: ['http://a.b.c', 'https://test-insight.bitpay.com:443'],
+      },
     },
-    testnet: {
-      provider: 'insight',
-      url: 'https://test-insight.bitpay.com:443',
-      // url: 'http://localhost:3001',
-      // Multiple servers (in priority order)
-      // url: ['http://a.b.c', 'https://test-insight.bitpay.com:443'],
+    bch: {
+      livenet: {
+        provider: 'insight',
+        url: 'https://cashexplorer.bitcoin.com',
```

Comment:
This is an external service; strongly recommend hosting on @BitPay's servers, but if not, please notify users of the risks before deploying this code to production.